docs/src/app/components/pages/components/LinearProgress/ExampleSimple.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import LinearProgress from 'material-ui/LinearProgress';
+import {LinearProgress} from 'material-ui';
```

Comment:
Sorry, that's written like this on purpose, have a look at [minimizing-bundle-size](https://material-ui-1dab0.firebaseapp.com/guides/minimizing-bundle-size) regarding the why. I think that it's the best default until tree shaking is better supported.